scalableminds/webknossos
# Finer Annotation visibility control

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- see different options in sharing modal
- try accessing a private annotation with other user from same organization => should not work

### Issues:
- fixes #3350 

------
